docs/source/release/v4.3.0/diffraction.rst

```diff
@@ -9,6 +9,10 @@ Diffraction Changes
     putting new features at the top of the section, followed by
     improvements, followed by bug fixes.
 
+New
+############
+- New loader for PANTHER instrument in diffraction mode.
```

Comment:
You should rather put a link to the loader, saying that it now supports also the diffraction mode. Note that the other instruments (IN4, IN5, IN6) as well can have the diffraction mode, but they almost never use it.